packages/scrubber-bar/test/scrubber-bar.test.js

```diff
@@ -93,11 +104,12 @@ describe('ScrubberBar', () => {
     const inputEvent = new Event('input');
 
     // we have to do this in a setTimeout so the event listener below has a chance to listen
-    setTimeout(() => { rangeSlider.dispatchEvent(inputEvent); });
+    setTimeout(() => {
+      rangeSlider.dispatchEvent(inputEvent);
+    });
```

Comment:
without a time set, is it just defaulting to `0`?

Reply:
Yeah with the scrubber bar, it defaults to a range of `0 - 100`, with a value of `0`, stepping by `0.1`.